test/unit/orchestration/pxe_suse_template

```diff
@@ -0,0 +1,4 @@
+DEFAULT linux
+LABEL linux
+KERNEL boot/OpenSuse-12.3-x86_64-linux
+APPEND initrd=boot/OpenSuse-12.3-x86_64-initrd ramdisk_size=65536 install=http://download.opensuse.org/distribution/12.3/repo/oss autoyast=http://ahost.com:3000/unattended/provision textmode=1
```

Comment:
I only see two more external files in this directory, how about moving this and the two others into Ruby HEREDOCs?